src/applications/mhv/secure-messaging/components/MessageThread/MessageThreadItem.jsx

```diff
@@ -23,104 +40,82 @@ const MessageThreadItem = props => {
-      if (e.keyCode === 32) {
-        e.preventDefault(); // prevent from scrolling to the footer
-      }
-
-      // expanding/collapsing on Enter or Space key press for accessibility
-      if (
-        (e.key === 'Enter' || e.key === ' ' || e.type === 'click') &&
-        e.target.tagName !== 'a' // prevent from collapsing when user clicks on a link in the message
-      ) {
-        setIsExpanded(!isExpanded);
-      }
-
-      if (!isExpanded) {
-        dispatch(markMessageAsReadInThread(message.messageId));
-      }
+  const handleExpand = isPreloaded => {
+    if (!isPreloaded) {
+      dispatch(markMessageAsReadInThread(messageId, isDraftThread));
     }
+    setIsExpanded(!isExpanded);
   };
 
-  const hasAttachments = useMemo(
-    () => {
-      return (
-        message.attachment ||
-        message.hasAttachments ||
-        message.attachments?.length
-      );
-    },
-    [message.attachment, message.hasAttachments, message.attachments],
-  );
+  // const header = () => {
+  //   return (
+  //     <>
+  //       <i
+  //         className="unread-icon fas fa-circle"
+  //         aria-hidden
+  //         style={{ visibility: isRead === true ? 'hidden' : '' }}
+  //       />
+  //       {dateFormat(sentDate, 'MMMM D, YYYY [at] h:mm a z')}
+  //     </>
+  //   );
+  // };
 
-  const ariaLabel = useMemo(
-    () => {
-      return `${!isRead ? 'New' : ''} message ${
-        hasAttachments ? 'with attachment' : ''
-      } from ${message.senderName}, ${dateFormat(
-        message.sentDate,
-        'MMMM D, YYYY [at] h:mm a z',
-      )}. Click to ${isExpanded ? 'Collapse message' : 'Expand message'}`;
-    },
-    [hasAttachments, isExpanded, isRead, message],
-  );
+  // const subheader = () => {
+  //   return (
+  //     <>
+  //       <i
+  //         className="fas fa-paperclip"
+  //         aria-hidden
+  //         style={{ visibility: attachments !== true ? 'hidden' : '' }}
+  //       />
+  //       {from}
+  //     </>
+  //   );
+  // };
 
-  return (
-    message && (
-      <>
-        <div className="older-message vads-u-padding-top--0p5 vads-u-padding-bottom--2 vads-u-display--flex vads-u-flex-direction--row">
-          <div
-            className="vads-u-flex--auto"
-            role="img"
-            aria-label={!isRead ? 'Unread message' : 'Previously read message'}
-          >
-            <i
-              className="unread-icon fas fa-circle"
-              aria-hidden
-              style={{ visibility: isRead === true ? 'hidden' : '' }}
-            />
-          </div>
+  // const ariaLabel = useMemo(
+  //   () => {
+  //     return `${!isRead ? 'New' : ''} message ${
+  //       attachment ? 'with attachment' : ''
+  //     } from ${senderName}, ${dateFormat(
+  //       sentDate,
+  //       'MMMM D, YYYY [at] h:mm a z',
+  //     )}. Click to ${isExpanded ? 'Collapse message' : 'Expand message'}`;
+  //   },
+  //   [attachment, isExpanded, isRead, senderName, sentDate],
+  // );

```

Comment:
Same for this one.. maybe put this in a helper file if we need this later.

Reply:
this is pending confirmation from UCD on what aria labels should be reading like once the accordion accepts icons.